webapp/src/components/backstage/select_users_below.tsx

```diff
@@ -23,53 +44,150 @@ const UserList = styled.div`
+    const permissionToEditMembers = useHasPlaybookPermission(PlaybookPermissionGeneral.ManageMembers, props.playbook);
+    const teamnameNameDisplaySetting = useSelector(getTeammateNameDisplaySetting) || '';
+    const users = useSelector(getUsers);
+
+    const handleAddUser = (userId: string) => {
+        props.onAddMember({user_id: userId, roles: [PlaybookRole.Member]});
+    };
+
+    const sortedMembers = props.members.slice().sort((a: PlaybookMember, b:PlaybookMember) => {
+        return displayUsername(users[a.user_id], teamnameNameDisplaySetting).localeCompare(displayUsername(users[b.user_id], teamnameNameDisplaySetting));
+    });
+
     return (
-        <ProfileAutocompleteContainer>
-            <ProfileAutocomplete
-                onAddUser={props.onAddUser}
-                userIds={props.userIds}
-                searchProfiles={props.searchProfiles}
-                getProfiles={props.getProfiles}
-            />
+        <Container>
+            {permissionToEditMembers &&
+            <ProfileAutocompleteContainer>
+                <ProfileAutocomplete
+                    onAddUser={handleAddUser}
+                    userIds={props.members.map((val: PlaybookMember) => val.user_id)}
+                    searchProfiles={props.searchProfiles}
+                    getProfiles={props.getProfiles}
+                />
+            </ProfileAutocompleteContainer>
+            }
             <UserList>
-                {props.userIds.map((userId) => (
+                {sortedMembers.map((member: PlaybookMember) => (
                     <UserLine
                         data-testid='user-line'
-                        key={userId}
-                    >
-                        <BelowLineProfile userId={userId}/>
-                        {
-                            props.userIds.length > 1 &&
-                            <RemoveLink onClick={() => props.onRemoveUser(userId)}>
-                                <FormattedMessage defaultMessage='Remove'/>
-                            </RemoveLink>
-                        }
-                    </UserLine>
+                        key={member.user_id}
+                        hasPermissionsToEditMembers={permissionToEditMembers}
+                        member={member}
+                        onRemoveUser={props.onRemoveUser}
+                        onMakeAdmin={props.onMakeAdmin}
+                        onMakeMember={props.onMakeMember}
+                    />
                 ))}
             </UserList>
-        </ProfileAutocompleteContainer>
+        </Container>
+    );
+};
+
+interface UserLineProps {
+    hasPermissionsToEditMembers: boolean
+    member: PlaybookMember
+    onRemoveUser: (userid: string) => void;
+    onMakeAdmin: (userid: string) => void;
+    onMakeMember: (userid: string) => void;
+}
+
+const MemberButton = styled.div`
+    display: inline-flex;
+    border-radius: 4px;
+    fill: rgb(var(--link-color-rgb));
+    color: rgb(var(--link-color-rgb));
```

Comment:
We can probably simplify this:
```suggestion
    fill: var(--link-color);
    color: var(--link-color);
```